cos-fleetshard-operator-debezium/src/main/kubernetes/kubernetes.yml

```diff
@@ -34,6 +34,7 @@ rules:
   - ""
   resources:
   - secrets
+  - configmaps
```

Comment:
What is correct with or without quotes?

Reply:
in this case it is the same (the other file is generated by k8s client so it depend on the yaml serdes setup)